src/baldr/curler.cc

```diff
@@ -6,7 +6,7 @@
 #include <stdexcept>
 #include <string>
 
-#ifdef CURL_STATICLIB
+#ifdef ENABLE_HTTP
```

Comment:
`CURL_STATICLIB` actually means to link a static library, I don't think we should use it to conditionally include this source code